server/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -415,21 +415,21 @@ public void testSelfJoinWithOrder() throws Exception {
 
     @Test
     public void test_self_join_with_order_and_limit_is_executed_with_qtf() throws Exception {
-        execute("create table doc.t (x int, y int)");
-        execute("insert into doc.t (x, y) values (1, 10), (2, 20)");
-        execute("refresh table doc.t");
-        execute("explain select * from doc.t as t1, doc.t as t2 order by t1.x, t2.x limit 3");
-        assertThat(printedTable(response.rows()), is(
+        execute("create table t (x int, y int)");
+        execute("insert into t (x, y) values (1, 10), (2, 20)");
+        execute("refresh table t");
+        execute("explain select * from t as t1, t as t2 order by t1.x, t2.x limit 3");
+        assertThat(printedTable(response.rows()).replace(sqlExecutor.getCurrentSchema() + ".", ""), is(
```

Comment:
Maybe create a  `printedTable()` which also takes the currentSchema and makes the replacement.